From: Roger Shimizu <rogershimizu@gmail.com>
Date: Sat, 16 Jul 2016 16:46:15 +0900
Subject: extend test timeout of bitset case to 120 seconds

It was easily got timeout on some low performance platforms, such as
armel, hppa, etc.
---
 tests/test-bitset.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-bitset.c b/tests/test-bitset.c
index 2476b69..c07386a 100644
--- a/tests/test-bitset.c
+++ b/tests/test-bitset.c
@@ -79,7 +79,7 @@ test_suite()
     Suite  *s = suite_create("bits");
 
     TCase  *tc_ds = tcase_create("bits");
-    tcase_set_timeout(tc_ds, 20.0);
+    tcase_set_timeout(tc_ds, 120.0);
     tcase_add_test(tc_ds, test_bitset);
     suite_add_tcase(s, tc_ds);
 
